src/plugins/discover/public/application/angular/directives/uninitialized.tsx

```diff
@@ -29,39 +29,30 @@ interface Props {
 export const DiscoverUninitialized = ({ onRefresh }: Props) => {
   return (
     <I18nProvider>
-      <EuiPage>
-        <EuiPageBody>
-          <EuiPageContent horizontalPosition="center">
-            <EuiEmptyPrompt
-              iconType="discoverApp"
-              title={
-                <h2>
-                  <FormattedMessage
-                    id="discover.uninitializedTitle"
-                    defaultMessage="Start searching"
-                  />
-                </h2>
-              }
-              body={
-                <p>
-                  <FormattedMessage
-                    id="discover.uninitializedText"
-                    defaultMessage="Write a query, add some filters, or simply hit Refresh to retrieve results for the current query."
-                  />
-                </p>
-              }
-              actions={
-                <EuiButton color="primary" fill onClick={onRefresh}>
-                  <FormattedMessage
-                    id="discover.uninitializedRefreshButtonText"
-                    defaultMessage="Refresh data"
-                  />
-                </EuiButton>
-              }
+      <EuiEmptyPrompt
+        iconType="discoverApp"
+        title={
+          <h2>
+            <FormattedMessage id="discover.uninitializedTitle" defaultMessage="Start searching" />
+          </h2>
+        }
+        body={
+          <p>
+            <FormattedMessage
+              id="discover.uninitializedText"
+              defaultMessage="Write a query, add some filters, or simply hit Refresh to retrieve results for the current query."
```

Comment:
The image in the PR shows an empty query bar, yet the text says "hit Refresh to retrieve results for the current query".  If the query is **always** for the default time period, could the text say that:

> Write a query, add some filters, or hit Refresh to view all updates in the last 15 minutes.

Reply:
It's not always the case: the default time period is configureable, also you might view an index pattern without time picker.